src/main/java/net/dungdealers/diapersplus/Diapers.java

```diff
@@ -12,5 +13,7 @@ public class Diapers implements ModInitializer {
 	@Override
 	public void onInitialize() {
 		LOGGER.info("poop");
+		DiaperItem.registerModItems();
```

Comment:
make sure you change this to DPItems.registerModItems() / whatever you change it to